docs/sdk/features/edgedb.md

```diff
@@ -0,0 +1,60 @@
+# EdgeDB Feature
+
+## Overview
+
+The EdgeDB feature allows the saving of user data into our EdgeDB storage and to be able to use the data in segmenting decisions without needing
+to pass that data in our SDKs every time. 
+
+## Usage
+
+### Rest API
+
+We are able to support updates to users using our EdgeDB Public Rest API. The docs for it can be found here. (Link to the completed API docs)
```

Comment:
we won't be able to merge this until this is changed to a link. You could just make a placeholder page for those docs for now and link it here?

Reply:
where would this placeholder page be hosted, `docs.devcycle.com` or `api.devcycle.com/docs`?

Reply:
I think the EdgeDB endpoints need to be documented as part of the client sdk api docs and the bucketing API docs since that's where the endpoints are